packages/ohm-js/test/test-ohm-syntax.js

```diff
@@ -1382,22 +1382,22 @@ describe('bootstrap', test => {
   const ns = ohm.grammars(ohmGrammarSource);
 
   test('it can recognize arithmetic grammar', t => {
-    assertSucceeds(t, ns.Ohm.match(arithmeticGrammarSource, 'Grammar'));
```

Comment:
Were these changes necessary, or is this an unrelated cleanup?

Reply:
Since we added new `Document` rule to grammar the default rule is now `Document`, I feel the tests should be validating against the new grammar and not a sub-rule (e.g. `Grammar`) of the main grammar. Hence the change and resulting fixes to array accesses below.